deployment/puppet/refinery/manifests/apache2.pp

```diff
@@ -32,10 +32,11 @@
   }
 
   class { 'apache::mod::wsgi':
-    mod_path     => 'mod_wsgi.so',
-    package_name => 'libapache2-mod-wsgi',
+    mod_path         => 'mod_wsgi.so',
+    package_name     => 'libapache2-mod-wsgi-py3',
+    wsgi_python_home => "${virtualenv}",
+    wsgi_python_path => "${django_root}",
```

Comment:
i will look into this - it might be touching a different .conf file than the apache::vhost but i can't remember.  i've looked at the puppet source code for this ( while going down the rabbit hole that led me to change these similar settings in apache::vhost) and it's very opaque so i would not be surprised if it were necessary.  also this is another thing to watch out for when we  upgrade to python3.8.  i don't think there a version of libapache2-mod-wsgi-py3 out for python3.8 so we'll need to switch over to pip-installing it - also i think the writer of the package suggests doing it this way going forward anyway which might explain its absence.  in any case, the way to do this was not simple to figure out, but i know how now :)

Reply:
i have removed these now - i appear to have forgotten reading this: https://stackoverflow.com/a/50165569

essentially, we have no choice but to download mod_wsgi from PyPi unless we want to have a broken version and nothing works